mmdet/models/dense_heads/anchor_head.py

```diff
@@ -400,6 +401,12 @@ def loss(self,
             concat_anchor_list.append(torch.cat(anchor_list[i]))
         all_anchor_list = images_to_levels(concat_anchor_list,
                                            num_level_anchors)
+        if self.loss_normalizer != -1:
+            avg_factor = (
```

Comment:
It seems that the implementation is different with that in [detectron2](https://github.com/facebookresearch/detectron2/blob/bf11a9b5bbc8e80cfe713c547203576a77a09065/detectron2/modeling/meta_arch/retinanet.py#L237). `self.loss_normalizer` is not updated here. Am i missing something?

Reply:
No, it seems that I write a bug here.

Reply:
Fixed